mcs_benchmark_data/cli/template_contexts/metadata_template.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional
+from dataclasses import dataclass
+from pathlib import Path
+
+from mcs_benchmark_data.path import ROOT_DIR_PATH
+from mcs_benchmark_data.cli.template_contexts.template import Template
+
+
+@dataclass(init=True)

```

Comment:
Not sure these templates really need to be `@dataclass`'s? Fine if the context is.